validators/fuji/schain-upgrade.md

```diff
@@ -64,59 +64,53 @@ Consensus: the build introduces BLAKE3 hash and consensus DB changes (for better
 
```

Comment:
this chain upgrade page is confusing because validators don't need to do half of the work here. Can we move the recharging wallet to a separate file and add it as a step in the fuji versions?

Reply:
May be let's remove  fuji/schain-upgrade.md and rename fuji-versions.md to something like fuji-instructions.md?

Reply:
Let's figure out later we should have a better process to reduce the duplicate work.